scheduler/main.go

```diff
@@ -2,19 +2,16 @@ package scheduler
 
 import (
 	"github.com/Sirupsen/logrus"
-	"github.com/spf13/viper"
 )
 
 // Create is used to build a new scheduler with default settings
-func Create() *eremeticScheduler {
-	return createEremeticScheduler(&Settings{
-		MaxQueueSize: viper.GetInt("queue_size"),
-	})
+func Create(settings *Settings) *eremeticScheduler {
+	return createEremeticScheduler(settings)
```

Comment:
both of these are just simple wrappers around `&eremeticScheduler{}` but, sure, we should remove a layer from this. What would be the best name to keep?